BigTuna/Base.lproj/Main.storyboard

```diff
@@ -35,6 +35,60 @@
             </objects>
             <point key="canvasLocation" x="-3650" y="824"/>
         </scene>
+        <!--Navigation Controller-->
+        <scene sceneID="P1a-6u-A96">
+            <objects>
+                <navigationController storyboardIdentifier="DiscoverViewController" id="z4k-cZ-esM" sceneMemberID="viewController">
+                    <tabBarItem key="tabBarItem" title="" image="DiscoverTabIcon" id="DiO-0G-bnb">
+                        <inset key="imageInsets" minX="0.0" minY="6" maxX="0.0" maxY="-6"/>
+                    </tabBarItem>
+                    <navigationBar key="navigationBar" contentMode="scaleToFill" insetsLayoutMarginsFromSafeArea="NO" id="V8T-ca-4qa">
+                        <rect key="frame" x="0.0" y="20" width="414" height="44"/>
+                        <autoresizingMask key="autoresizingMask"/>
+                    </navigationBar>
+                    <connections>
+                        <segue destination="CUp-l1-Gza" kind="relationship" relationship="rootViewController" id="bwg-rq-o94"/>
+                    </connections>
+                </navigationController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="a9J-3y-Ve8" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="-2659" y="-194"/>
+        </scene>
+        <!--Discover View Controller-->
+        <scene sceneID="TpZ-1R-sLc">
+            <objects>
+                <collectionViewController id="CUp-l1-Gza" customClass="DiscoverViewController" customModule="BigTuna" customModuleProvider="target" sceneMemberID="viewController">
+                    <collectionView key="view" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" dataMode="prototypes" id="0p1-Tx-f3L">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="736"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                        <collectionViewFlowLayout key="collectionViewLayout" minimumLineSpacing="10" minimumInteritemSpacing="10" id="vhl-sn-oDL">
```

Comment:
are we still adding things and using the storyboard?

Reply:
We are getting off the Storyboard but because the Tab Bar Controller is still on it, any tabs created apparently also need to be on the Storyboard; Discovery page had to be added to Storyboard to be linked to the tab bar controller. 😞